qiskit/aqua/operators/expectation_values/projector_overlap.py

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8 -*-
+
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2020.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+""" Projector Overlap Expectation Value algorithm.  """
+
+import logging
+from typing import Union
+import numpy as np
+
+from qiskit.providers import BaseBackend
+
+from ..operator_base import OperatorBase
+from .expectation_base import ExpectationBase
+
+logger = logging.getLogger(__name__)
+
+
+class ProjectorOverlap(ExpectationBase):
```

Comment:
Why is this called projector overlap? What exactly is it supposed to do? Also since it is not implemented we could postpone the addition of this algorithm, if that helps in cleaning up the rest 🙂

Reply:
Deleted